cmd/server-main.go

```diff
@@ -306,6 +277,9 @@ func serverMain(ctx *cli.Context) {
 	// Handle all server environment vars.
 	serverHandleEnvVars()
 
+	// Set node name, only set for distributed setup.
```

Comment:
Should there be a check for distributed mode?

Reply:
Its internal @kannappanr